ships/botany_bay.json

```diff
@@ -0,0 +1,3008 @@
+{
+  "description": "Much like the crew that operated it, the Botany Bay is a relic from humanity\u2019s distant past. Built on Earth in the late 20th century, it was intended to be humanity\u2019s foray into deep space exploration. The vessel\u2019s engines utilized revolutionary nuclear fission technology that was believed to be a completely self-sufficient energy source and its X-25 network array could gather data at an astounding rate.",
+  "blueprints": {
+    "cost": 0,
+    "quantity": 60
+  },
+  "build_costs": {
+    "materials": [
+      {
+        "type": "gas",
+        "grade": 3,
+        "rarity": "uncommon",
+        "value": 5
+      },
+      {
+        "type": "gas",
+        "grade": 3,
+        "rarity": "rare",
+        "value": 9
+      },
+      {
+        "type": "ore",
+        "grade": 3,
+        "rarity": "uncommon",
+        "value": 18
+      },
+      {
+        "type": "ore",
+        "grade": 3,
+        "rarity": "rare",
+        "value": 39
+      }
+    ],
+    "others": [
+      {
+        "type": "3\u2605\u00a0Survey\u00a0Parts",
+        "value": 1510
+      }
+    ],
+    "resources": [
+      {
+        "type": "tritanium",
+        "value": 4155000
+      },
+      {
+        "type": "dilithium",
+        "value": 176500
+      }
+    ]
+  },
+  "faction": "neutral",
+  "grade": 3,
+  "img_url": "./assets/botany_bay.png",
+  "rarity": "uncommon",
+  "requirements": [
+    {
+      "name": "Shipyard",
+      "level": 18
+    }
+  ],
+  "scrap_info": {
+    "scrapable": false
+  },
+  "ship_ability": {
+    "title": "Data Mining",
+    "description": "",
```

Comment:
Do we have these descriptions somewhere, or do we need to ask the comunity for imput?